### PR TITLE
Update pull request template

### PR DESCRIPTION
**Pull request by Panquesito7**
**Description**

- Update pull request template with more information and proper grammar.

This PR fixes #

**Notes for Reviewers:**

**[Signed commits](https://github.com/DevsTrons/hello-world/blob/master/CONTRIBUTING.md)**

- [x] Yes, I am a Member of **DEVSTRONS'**

**Discord id** (eg: DEVSTRONS'#0238):

- Panquesito7#3723

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,11 +13,11 @@ This PR fixes #
 
 - [ ] Yes, I am a Member of **DEVSTRONS'**
 
-**Discord id** (eg: DEVSTRONS'#0238):
-<!-- Enter your Discord's ID -->
+**Discord ID** (e.g.: DEVSTRONS'#0238)
 
+- Your Discord ID <!-- Enter your Discord ID -->
```
Comment (amino19) [inline .github/pull_request_template.md:18]: @Panquesito7: I already gave the example, no need for this extension of keywords.
```suggestion
<!-- Enter your Discord ID -->
```

**Review by amino19 — CHANGES_REQUESTED** [review]

**Comment by amino19** [comment]
@vsnthdev: Pl request further changes to Pull Request template as you said, you have an idea!
Let's work on it now! :rocket:

**Comment by vsnthdev** [comment]
We can use GitHub Form Schema to improve things for the PR template. Read [more about this here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#about-githubs-form-schema). I'll be sending you a PR with the new issue template soon.

**Comment by Panquesito7** [comment]
> We can use GitHub Form Schema to improve things for the PR template. Read [more about this here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#about-githubs-form-schema). I'll be sending you a PR with the new issue template soon.

AFAIK, forms cannot be used in pull request templates.

**Comment by vsnthdev** [comment]
Yup, that looks good to me 👍

**Pull request merged**